docs/bugpattern/UsageCheckers.md

```diff
@@ -0,0 +1,5 @@
+###UnusedInject
```

Comment:
I think this should be `UnusedMemberInjection` since this doesn't cover constructor injection (which is probably the most common form of injection outside of android projects).